scvi/models/__init__.py

```diff
@@ -1,18 +1,8 @@
-from .classifier import Classifier
-from .scanvi import SCANVI
-from .vae import VAE, LDVAE
-from .autozivae import AutoZIVAE
-from .vaec import VAEC
-from .jvae import JVAE
+from .scvi import SCVI
 from .totalvi import TOTALVI
+from .linear_scvi import LinearSCVI
```

Comment:
why not TotalVI, LinearScVI, AutoZI, ScANVI, GimVI, ScVI?

or LINEARSCVI but it sounds funny

Reply:
Haha, yea LINEARSCVI might be a bit much, but I think we can keep it as is for now. Seems consistent with sklearn where they have PCA, TSNE, but IncrementalPCA.